src/hospitals/hospitals.graphql

```diff
@@ -1,6 +1,7 @@
 # ---------------------------------- Querys ---------------------------------- #
 type Query {
-  hospitals: [Hospital]!
+  hospitals(filter: String!): [Hospital]!
```

Comment:
as filter not described in task, make it not required, because on FE we have to provide it anytime we want use query